arm-operationalinsights/compositeOperationalInsights.json

```diff
@@ -0,0 +1,11 @@
+{
+  "info": {
+    "title": "OperationalInsightsManagementClient",
+    "description": ".Net client wrapper for the REST API for Azure Operational Insights"
```

Comment:
should be **Azure Log Analytics**, although our RP still have old name OperationalInsights